python_modules/libraries/dagster-duckdb/dagster_duckdb/io_manager.py

```diff
@@ -262,6 +262,14 @@ def delete_table_slice(context: OutputContext, table_slice: TableSlice, connecti
         except duckdb.CatalogException:
             # table doesn't exist yet, so ignore the error
             pass
+        except duckdb.BinderException as ex:
```

Comment:
I think adding a test for this behavior would be helpful! That way if there are changes in the duckdb package that break this path we'll find out sooner. I think the easiest place for you to put this would be in `python_modules/libraries/dagster-duckdb-pandas/dagster_duckdb_pandas_tests/test_type_handler.py`